dlc-messages/src/lib.rs

```diff
@@ -86,6 +86,7 @@ impl_type!(RENEW_CHANNEL_OFFER_TYPE, RenewOffer, 43014);
 impl_type!(RENEW_CHANNEL_ACCEPT_TYPE, RenewAccept, 43016);
 impl_type!(RENEW_CHANNEL_CONFIRM_TYPE, RenewConfirm, 43018);
 impl_type!(RENEW_CHANNEL_FINALIZE_TYPE, RenewFinalize, 43020);
+impl_type!(RENEW_CHANNEL_REVOKE_TYPE, RenewRevoke, 43024);
```

Comment:
Doesn't this one have the same number as the one on [line 95](https://github.com/p2pderivatives/rust-dlc/pull/116/files#diff-ff7ace341c38836f8a04c27ae2f46240efb6ab702d3fafe383199ede4650f3adR95)?

Reply:
Ouups fixed thanks! It'd be nice to have something to check this kind of errors...